rholang/examples/intdupe.rho

```diff
@@ -0,0 +1,61 @@
+// This code is designed to create a large number of comm events
+// For testing purposes. It strives to make the comm events on as wide
+// a range of channels as possible, but because rholang lacks explicit
+// substitution, some channels are bottlenecks.
+//
+// The technique is to create a single listen and a single send on each
+// integer channel up to some upper bound. The technique for creating the
+// sends and listens is illustrated by this tree. treeHelper essentially
+// takes an integer, produces a comm over the public channel represented
+// by that integer, and then does likewise for each of it's children.
+//
+// Notice that given a particular node, it's two children can be calculated
+// as 2n and 2n+1
+//
+//           1
+//        /      \
+//      2          3
+//    /   \      /   \
+//   4     5    6     7
+//
+// When run with a parameter of n, 2^n + 2^(n-1) + 1 comm events will be created
+// Here is the breakdown
+// 2^n non-contentious comms over the integer channels
+// 2^(n-1) bottlenecked comms over treeHelper
+// 1 comm over buildTree
+
+new buildTree,
+    treeHelper,
+    stdout(`rho:io:stdout`) in {
+
+  /**
+   * Users can customize how many comms are desired by changing this line.
+   * Roughly 2^(n + 1) comms will occur. Use -1 for essentially unlimited comms.
+   * Other lines will not need to be changed by end users.
+   */
+  buildTree!(10) |
```

Comment:
Can we make the default 5 for now? We use random contracts from the examples folder for testnet right now and I don't want to heavily bias it towards stress testing contracts as of now. Of course in the future we probably want to pick out the contracts that we think would be more representative of what the real network would be running but I think that might be a couple weeks down the line.

Reply:
I think this contract should go under `https://github.com/rchain/perf-harness/tree/master/contracts` @JoshOrndorff . If you opened a PR against that repo we could check right away what #COMMs is being generated

Reply:
Sure, 5 sounds fine.